.drone.yml

```diff
@@ -36,6 +36,21 @@ steps:
       exclude:
       - pull_request
 
+- name: publish
+  image: maven:3.8-openjdk-17
+  depends_on:
+  - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+      from_secret: artifactory_token
+  commands:
+  - mvn clean install
+  - mvn -s ./jparest_settings.xml -f pom.xml install -B org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy -DuniqueVersion=false -DaltReleaseDeploymentRepository=artifactory::default::https://artifactory.digital.homeoffice.gov.uk/artifactory/callisto-jparest -DskipTests
```

Comment:
Is this needed altReleaseDeploymentRepository?
If the id in the jparest_settings.xml was set to match the repositories in the pom.xml changes then I don't think it's required

Reply:
This is correct. I have removed it